flexget/components/sites/sites/piratebay.py

```diff
@@ -130,76 +137,80 @@ def search(self, task, entry, config=None):
         if not isinstance(config, dict):
             config = {}
         self.set_urls(config.get('url', URL))
-        sort = SORT.get(config.get('sort_by', 'seeds'))
-        if config.get('sort_reverse'):
-            sort += 1
+        sort = SORT.get(config.get('sort_by', 'default'))
+        sort_reverse = bool(config.get('sort_reverse', 'True'))
         if isinstance(config.get('category'), int):
             category = config['category']
         else:
             category = CATEGORIES.get(config.get('category', 'all'))
-        filter_url = '/0/%d/%d' % (sort, category)
+        filter_url = '&cat=%s' % category
 
         entries = set()
         for search_string in entry.get('search_strings', [entry['title']]):
             query = normalize_unicode(search_string)
 
             # TPB search doesn't like dashes or quotes
-            query = query.replace('-', ' ').replace("'", " ")
+            # query = query.replace('-', ' ').replace("'", " ")
 
             # urllib.quote will crash if the unicode string has non ascii characters, so encode in utf-8 beforehand
-            url = '%s/search/%s%s' % (self.url, quote(query.encode('utf-8')), filter_url)
+            url = '%s/q.php?q=%s%s' % (self.url, quote(query.encode('utf-8')), filter_url)
```

Comment:
please pass url params as params dict and it to the request. That way you also don't need to bother about url quoting it yourself